skydb.go

```diff
@@ -0,0 +1,106 @@
+package skynet
+
+import (
+	"encoding/hex"
+	"fmt"
+	"gitlab.com/NebulousLabs/errors"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+// maxRevision is the maximum allowed value for an entry revision.
+// Setting an entry revision to this value prevents it from being updated further.
+const maxRevision uint64 = 18446744073709551615
```

Comment:
`math.MaxUint64` is a golang constant. No need to define it here.

Reply:
Thx for the catch! https://github.com/kamy22/go-skynet/pull/1/commits/794bc4ab3106287d8834f077007c0bb44b901b86 ;)